crates/vm/levm/tests/tests.rs

```diff
@@ -4643,3 +4646,50 @@ fn ecpairing_test() {
         (ECPAIRING_BASE_COST + ECPAIRING_GROUP_COST * 2)
     );
 }
+
+#[test]
+fn blake2f_test() {
+    let calldata = hex::decode("0000000c48c9bdf267e6096a3ba7ca8485ae67bb2bf894fe72f36e3cf1361d5f3af54fa5d182e6ad7f520e511f6c3e2b8c68059b6bbd41fbabd9831f79217e1319cde05b61626300000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000300000000000000000000000000000001").unwrap();
+    let calldata = Bytes::from(calldata);
+
+    let mut consumed_gas = 0;
+    let result = blake2f(&calldata, 10000, &mut consumed_gas).unwrap();
+
+    let expected_result = Bytes::from(hex::decode("ba80a53f981c4d0d6a2797b69f12f6e94c212f14685ac4b74b12bb6fdbffa2d17d87c5392aab792dc252d5de4533cc9518d38aa8dbf1925ab92386edd4009923").unwrap());
+
+    assert_eq!(result, expected_result);
+    assert_eq!(consumed_gas, 12 * BLAKE2F_ROUND_COST);
+}
+
+/*
+#[test]
+fn blake2f_test_2() {
```

Comment:
Ditto
```suggestion
#[test]
// Source: https://eips.ethereum.org/EIPS/eip-152#test-vector-2
fn blake2f_test_2() {
```

Reply:
"Done" in [Add source of second test](https://github.com/lambdaclass/ethrex/pull/1563/commits/c5a6d26a06c5bbdba26494dc87f01e96935f11c1)